pkg/opensearch/response_parser.go

```diff
@@ -265,6 +271,56 @@ func processTraceSpansResponse(res *es.SearchResponse, queryRes backend.DataResp
 	queryRes.Frames = data.Frames{frame}
 	return queryRes
 }
+
+func processTraceListResponse(res *es.SearchResponse, dsUID string, dsName string, queryRes backend.DataResponse) backend.DataResponse {
+	// trace list queries are hardcoded with a fairly hardcoded response format
+	// but es.SearchResponse is deliberately not typed as in other query cases it can be much more open ended
+	rawTraces := res.Aggregations["traces"].(map[string]interface{})["buckets"].([]interface{})
+
+	// get values from raw traces response
+	traceIds := []string{}
+	traceGroups := []string{}
+	traceLatencies := []float64{}
+	traceErrorCounts := []float64{}
+	traceLastUpdated := []time.Time{}
+	for _, trace := range rawTraces {
```

Comment:
Nit: I think this would be a little clearer if `t` and `trace` swapped names. Or if `t` had a more descriptive name